### PR TITLE
resources rename

### PR DESCRIPTION
**Pull request by tsebastiani**
- Renamed all resources and references from Openspot-NG to CRC-Cloud  

- Added the new container image reference on crcont org on quay based on centos stream 9.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-    apk add --no-cache jq && \
-    apk add --no-cache netcat-openbsd && \
-    apk add --no-cache bash && \
-    apk add --no-cache sed && \ 
-    apk add --no-cache curl && \
-    apk add --no-cache figlet && \
-    apk add --no-cache openssh-client-default 
+ENV PATH $PATH:/root/.awscliv2/binaries
+RUN rpm -Uvh https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm &&\
+    dnf install -y which &&\
```
Comment (lmilbaum) [inline Dockerfile:5]: Do you mind consolidating the dnf commands into one?

File: VERSION [hunk 2]
```diff
@@ -0,0 +1 @@
+v1.0
```
Comment (lmilbaum) [inline VERSION:1]: Please add a newline

**Review by lmilbaum — CHANGES_REQUESTED** [review]

File: VERSION (hunk 2 above)
Comment (tsebastiani) [reply VERSION:1]: sorry, still have to configure pre-commit hooks -.-'

File: Dockerfile (hunk 1 above)
Comment (tsebastiani) [reply Dockerfile:5]: Forgot your previous suggestion, my fault 👍🏻

**Pull request merged**